inc/utils.php

```diff
@@ -44,7 +44,7 @@ function mollieWooCommerceComponentsStylesForAvailableGateways()
  */
 function mollieWooCommerceCardToken()
 {
-    return $cardToken = filter_input(INPUT_POST, 'cardToken', FILTER_SANITIZE_STRING) ?: '';
+    return $cardToken = filter_input(INPUT_POST, 'cardToken', FILTER_SANITIZE_SPECIAL_CHARS) ?: '';
```

Comment:
I thought it was supposed to be replaced by ` wc_clean( wp_unslash(`?

Reply:
I don't think there is an agreement to use only `wc_clean( wp_unslash(` I will change them all if is required

Reply:
I just wonder what we should use :D 

`SANITIZE_SPECIAL_CHARS` probably does less things.